src/FormData.ts

```
@@ -0,0 +1,32 @@
+    let filename;
+    let contentDisposition;
+
+    if (typeof options.filepath === 'string') {
+      // custom filepath for relative paths
+      filename = path.normalize(options.filepath).replace(/\\/g, '/');
+    } else if (options.filename || value.name || value.path) {
+      // custom filename take precedence
+      // formidable and the browser add a name property
+      // fs- and request- streams have path property
+      filename = path.basename(options.filename || value.name || value.path);
+    } else if (value.readable && value.hasOwnProperty('httpVersion')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)